### PR TITLE
feat: add new sync status icons

### PR DESCRIPTION
**Pull request by ryanwolhuter**
use new sync status icons for drives
removed pnpm lockfile and replaced with yarn

### CODE REVIEW

File: src/hooks/useDrivesContainer.ts [hunk 1]
```diff
@@ -31,30 +38,46 @@ export function getNodePath(node: Node, allNodes: Node[]): string {
...
-export function driveToBaseItems(
-    drive: DocumentDriveDocument,
-): Array<BaseTreeItem> {
+async function getSyncStatus(
+    driveId: string,
+    type: DriveType,
+): Promise<SyncStatus | undefined> {
+    if (getIsLocalDrive(type)) return;
+    try {
+        (await window.electronAPI?.documentDrive.getSyncStatus(
```
Comment (acaldas) [inline src/hooks/useDrivesContainer.ts:58]: This should be called using `useDocumentDriveServer`. `electronAPI` is not defined when Connect is run on the browser.
`useDocumentDriveServer` needs to provide a `getSyncStatus` method.

**Review by acaldas — CHANGES_REQUESTED** [review]

File: src/hooks/useDrivesContainer.ts (hunk 1 above)
Comment (ryanwolhuter) [reply src/hooks/useDrivesContainer.ts:58]: ok cool will refactor

File: src/hooks/useDocumentDriveServer.ts [hunk 2]
```diff
@@ -341,6 +347,20 @@ export function useDocumentDriveServer(
         );
     }
 
+    async function getSyncStatus(
+        driveId: string,
+        type: DriveType,
+    ): Promise<SyncStatus | undefined> {
+        if (getIsLocalDrive(type)) return;
+        try {
+            server.getSyncStatus(driveId);
```
Comment (acaldas) [inline src/hooks/useDocumentDriveServer.ts:357]: If `server.getSyncStatus` returns a status it is not being returned right? Unless I'm misunderstanding the code.

Comment (ryanwolhuter) [reply src/hooks/useDocumentDriveServer.ts]: woops yeah I must have somehow lost my return statement when I was refactoring.

fixed and pushed now.

**Review by acaldas — APPROVED** [review]

**Pull request merged**